smpte20xx/transport/segment.go

```diff
@@ -0,0 +1,140 @@
+package transport
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/Comcast/gots/v2/packet"
+	"github.com/eluv-io/avpipe/goavpipe"
+)
+
+type SequentialOpener interface {
+	OpenNext(fd int64) (io.WriteCloser, error)
+}
```

Comment:
Should we move this file under `broadcastproto/transport`?  And we can just keep the seqOpener and remove the test file outputs (we can reconstruct the test files in the CLI using a test seqOpener)

Reply:
Ah, I declared the interface in `broadcastproto/mpegts` but missed swapping all dependencies on it.